modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector-reference.adoc

```diff
@@ -3,283 +3,338 @@ ifndef::env-site,env-github[]
-| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment will fail if the test doesn't pass after exhausting the associated reconnection strategy |  |
-| Consumer Key a| String |  The OAuth consumerKey as registered with the service provider |  | x
-| Consumer Secret a| String |  The OAuth consumerSecret as registered with the service provider |  | x
-| Authorization Url a| String |  The service provider's authorization endpoint URL |  `+https://login.microsoftonline.com/oauth2/authorize+` |
-| Access Token Url a| String |  The service provider's accessToken endpoint URL |  `+https://login.microsoftonline.com/oauth2/token+` |
-| Scopes a| String |  The OAuth scopes to be requested during the dance. If not provided, it will default to those in the annotation |  |
-| Resource Owner Id a| String |  The resourceOwnerId which each component should use if it doesn't reference otherwise. |  |
-| Before a| String |  The name of a flow to be executed right before starting the OAuth dance |  |
-| After a| String |  The name of a flow to be executed right after an accessToken has been received |  |
-| Listener Config a| String |  A reference to a `+<http:listener-config />+` to be used in order to create the listener that will catch the access token callback endpoint. |  | x
-| Callback Path a| String |  The path of the access token callback endpoint |  | x
-| Authorize Path a| String |  The path of the local HTTP endpoint which triggers the OAuth dance |  | x
-| External Callback Url a| String |  If the callback endpoint is behind a proxy or should be accessed through a non direct URL, use this parameter to tell the OAuth provider the URL it should use to access the callback |  |
-| Object Store a| String |  A reference to the object store that should be used to store each resource owner id's data. If not specified, runtime will automatically provision the default one. |  |
+| Response Type a| String |  |  +++query+++ | 
+| Read Timeout a| Number |  +++Specifies the amount of time, in milliseconds, that the consumer waits for a response before it times out. Default value is 0, which means infinite.+++ |  +++0+++ | 
+| Connection Timeout a| Number |  +++Specifies the amount of time, in milliseconds, that the consumer will attempt to establish a connection before it times out. Default value is 0, which means infinite.+++ |  +++0+++ | 
+| Host a| String |  +++Hostname of the proxy. If this property is not set then no proxy will be used, otherwise a proxy will be used, but a proxy host must be specified.+++ |  | 
+| Port a| Number |  +++Port of the proxy. If host is set then this property must be set and cannot be a negative number.+++ |  | 
+| Username a| String |  +++Username used to authenticate against the proxy. If this property is not set then no authentication is going to use against the proxy, otherwise this value must be specified.+++ |  | 
```

Comment:
Is there a way to reword this?: If this property is not set then no authentication is going to use against the proxy, otherwise this value must be specified.
Wondering if 'going to use against' might be a bit confusing?
Ignore if not appropriate.